lib/formtools/plugins/plugins-helpers.js

```diff
@@ -252,6 +242,50 @@ var filtersDefault = function (filters) {
 
 };
 
+var exportDefault = function (exports) {
+
+    if (exports === undefined || exports === null) {
+        return undefined;
```

Comment:
Sorry. I didn't realise it would have that affect. I guess that is why you don't do it! This will help in the future though:

```
git fetch
git reset --hard origin/{branch-name-or-master}
```